services/openshift/scripts/openshift_provision

```diff
@@ -43,23 +43,16 @@ done
 # Create Docker Registry
 if [ ! -f ${ORIGIN_DIR}/configured.registry ]; then
   echo "[INFO] Configuring Docker Registry"
-  oadm registry --create --credentials=${OPENSHIFT_DIR}/openshift-registry.kubeconfig || exit 1
+  oadm registry --create --service-account=registry || exit 1
```

Comment:
@praveenkumar can you please add some comments here, so that it would be easy to read the code when compared to https://docs.openshift.org/latest/install_config/install/docker_registry.html#registry-non-production-use